docker-compose.yml

```diff
@@ -2,43 +2,32 @@ version: "3"
 
 services:
   db:
-    image: postgres:13
+    image: postgres:latest
     env_file:
-      # Use template env file ./docker/.env.dev.sample
-      #   and configure your container variables
-      - ./docker/.env
-
+      - .env
     volumes:
       - postgres_data:/var/lib/postgresql/data
 
   app:
     build:
       context: .
-      args:
-        # overriding with sample .env development file for container build,
-        # but make sure you have your own .env file for development
-        DOTENV: .env.dev.sample
     image: anemperfeina:1.0
-
+    user: root
     depends_on:
       - db
-
     env_file:
-      # Use template env file ./docker/.env.dev.sample
```

Comment:
Remember to delete `docker/.env.dev.sample` file :)

Reply:
Done